### PR TITLE
new file added

### PR DESCRIPTION
**Pull request by awsdevopskiran**
file under review

### CODE REVIEW

File: file4.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 this b1 file
+file created in bew branch v1
```
Comment (swapnil-aws2022) [inline file4.txt:2]: add proper comment

**Review by swapnil-aws2022 — CHANGES_REQUESTED** [review]
kindly check file4 review

File: file4.txt (hunk 1 above)
Comment (awsdevopskiran) [reply file4.txt:2]: comment added properly

Comment (swapnil-aws2022) [reply file4.txt:2]: comment accepted

**Review by swapnil-aws2022 — APPROVED** [review]
resolved

**Pull request merged**